### PR TITLE
Fix 6.8.0 release adding missing class alises

### LINKED ISSUE #1112: elasticsearch/elasticsearch 6.8.0 removes some classes, and doesn't leave aliases behind
### Summary of problem or feature request

Downstream tasks: https://phabricator.wikimedia.org/T276854, https://phabricator.wikimedia.org/T276320 and https://github.com/ruflin/Elastica/issues/1908

Mentioned on https://github.com/elastic/elasticsearch-php/commit/2da5f9337a269113954a84fe8e79ea6843f0e7c3

In 6.8, at least `Elasticsearch\Endpoints\Indices\Aliases\Update` disappeared. Therefore trying to instantiate it fails: `Error from line 451 of /var/www/html/w/extensions/Elastica/vendor/ruflin/elastica/lib/Elastica/Index.php: Class 'Elasticsearch\Endpoints\Indices\Aliases\Update' not found`

 There is seemingly supposed to be an alias left behind, but that doesn't seem to have happened. Other classes seemingly disappeared too, without aliases being left.

### Code snippet of problem

https://github.com/ruflin/Elastica/blob/6.x/lib/Elastica/Index.php#L451

```php
use Elasticsearch\Endpoints\Indices\Aliases\Update;

$endpoint = new Update();
```

### System details

- Operating System: n/a
- PHP Version: n/a
- ES-PHP client version: 6.8.0
- Elasticsearch version: n/a

### PR DESCRIPTION
**Pull request by ezimuel**
This PR fixes #1112 adding the following missing class aliases for 6.7 BC:
- `Elasticsearch\Endpoints\Indices\Aliases\Update`
- `Elasticsearch\Endpoints\Indices\Exists\Types`
- `Elasticsearch\Endpoints\Indices\Field\Get`

Moreover, it adds a missing function `getAliases()` (proxy of `getAlias()`) in `Elasticsearch\Namespaces\IndicesNamespace`.

### CODE REVIEW

**Comment by ezimuel** [comment]
@reedy, @phroggar, @ruflin this should fix the 6.8.0 issue. Let me know if you review, thanks!

File: src/autoload.php [hunk 1]
```diff
@@ -19,47 +19,50 @@
...
+    '\Elasticsearch\Endpoints\Ingest\Pipeline\ProcessorGrok' => '\Elasticsearch\Endpoints\Ingest\ProcessorGrok',
+    '\Elasticsearch\Endpoints\Script\Get' => '\Elasticsearch\Endpoints\GetScript',
+    '\Elasticsearch\Endpoints\Script\Put' => '\Elasticsearch\Endpoints\PutScript',
+    '\Elasticsearch\Endpoints\Script\Delete' => '\Elasticsearch\Endpoints\DeleteScript',
+    '\Elasticsearch\Endpoints\Snapshot\Repository\Create' => '\Elasticsearch\Endpoints\Snapshot\CreateRepository',
+    '\Elasticsearch\Endpoints\Snapshot\Repository\Delete' => '\Elasticsearch\Endpoints\Snapshot\DeleteRepository',
+    '\Elasticsearch\Endpoints\Snapshot\Repository\Get' => '\Elasticsearch\Endpoints\Snapshot\GetRepository',
+    '\Elasticsearch\Endpoints\Snapshot\Repository\Verify' => '\Elasticsearch\Endpoints\Snapshot\VerifyRepository',
+    '\Elasticsearch\Endpoints\Source\Get' => '\Elasticsearch\Endpoints\GetSource',
+    '\Elasticsearch\Endpoints\Tasks\TasksList' => '\Elasticsearch\Endpoints\Tasks\ListTasks'
```
Comment (reedy) [inline src/autoload.php:62]: I note the whitespace changes make this a little harder to review :)

**Review by ruflin — APPROVED** [review]
LGTM. 

@reedy Any chance you could try this on your end if it fixes all the issues?

File: src/autoload.php (hunk 1 above)
Comment (ezimuel) [reply src/autoload.php]: I fixed the indentation.

**Review by reedy — APPROVED** [review]
Our CI stack seems happy with this change :)

**Pull request merged**